--- lirc-xmms-plugin-1.2/acinclude.m4.orig	Mon Sep 30 21:54:46 2002
+++ lirc-xmms-plugin-1.2/acinclude.m4	Mon Sep 30 21:54:46 2002
@@ -11,8 +11,11 @@
 have_viavoice_tts=yes
 AC_REQUIRE_CPP
 AC_CHECK_LIB(ibmeci50,eciNew,
-AC_CHECK_LIB(pthread,main,
-AC_CHECK_HEADER(eci.h,true,have_viavoice_tts=no),have_viavoice_tts=no),have_viavoice_tts=no)
+  [AC_CHECK_LIB(pthread,main,
+    [AC_CHECK_HEADER(eci.h,true,[have_viavoice_tts=no])],[have_viavoice_tts=no])
+  ],
+  [have_viavoice_tts=no]
+)
 
 if test "$have_viavoice_tts" = "yes"; then
   AC_DEFINE(HAVE_VIAVOICE_TTS)
